app/proto/auth.proto

```diff
@@ -184,10 +184,13 @@ message ConfirmChangeEmailReq{
   string change_email_token = 1 [ (sensitive) = true ];
 }
 
+enum EmailConfirmationState {
+  EMAIL_CONFIRMATION_STATE_UNKNOWN = 0;
```

Comment:
Is this ever used/when would this be used?

Reply:
It would show up if the value is missing, which is recommended in the protobuf docs. Protocol buffers don't track presence, so this allows determining if this field is missing

Reply:
It's automatic? Got it.